### PR TITLE
fix: paramter type error

### PR DESCRIPTION
**Pull request by fireairforce**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/22040

### CODE REVIEW

File: components/tree/Tree.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export interface AntTreeNodeProps {
   selected?: boolean;
   selectable?: boolean;
   icon?: ((treeNode: AntdTreeNodeAttribute) => React.ReactNode) | React.ReactNode;
-  children?: React.ReactNode;
+  children?: React.ReactNode[];
```
Comment (zombieJ) [inline components/tree/Tree.tsx:46]: 截屏2020-05-08 上午11 05 50

ReactNode already contains `ReactNode[]`. No need to modify this.

**Comment by zombieJ** [comment]
Seem your patch v4 branch to fix v3 issue. Please double confirm.

**Comment by fireairforce** [comment]
ok. i will close this~

File: components/tree/Tree.tsx (hunk 1 above)
Comment (fireairforce) [reply components/tree/Tree.tsx:46]: ok~

Comment (fireairforce) [reply components/tree/Tree.tsx:46]: In fact, here is a Object Type?

**Pull request closed without merging**